cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -198,17 +202,15 @@ func start(log log.Component, config config.Component, taggerComp tagger.Compone
 
 	// Initialize remote configuration
 	var rcClient *rcclient.Client
-	var rcService *rcservice.Service
-	if pkgconfig.IsRemoteConfigEnabled(pkgconfig.Datadog) {
+	rcService, isSet := rcServiceOptional.Get()
```

Comment:
We tend not to use "Optional" in variable names for optional types. Instead, I would recommend replacing `rcServiceOptional` with `rcService` and `rcService` with `rcserv` as a contraction is how this naming is usually handled (when an optional and a concrete type appear in the same function).